### DISCUSSION TITLE
Can't log in: error 500

### DISCUSSION THREAD

**Discussion by acid-bong** [community/community#120432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Expected behaviour: after I enter my username and password GitHub should ask me for the 2FA and let me in. 

Actual behaviour: Github redirects me to https://github.com/session, which displays error 500. Refreshing the page doesn't help.

How do i proceed?

**Comment by acid-bong** [comment]
New detail: turns out I can login while using another country's proxy/VPN (I'm from Moldova, can successfully log in via a VPN server in the Netherlands)

**Reply by filin2009** [reply]
Same problem in Moldova (Tiraspol) - just logined only using VPN (United Kingdom)
